Makefile

```diff
@@ -1,14 +1,12 @@
 .DEFAULT_GOAL := help
 SHELL=/bin/bash
 APP_DIR=tests/Application
-SYLIUS_VERSION=1.12.0
-SYLIUS_PAYPAL_VERSION=1.5.0
+SYLIUS_VERSION=1.13.0
 SYMFONY=cd ${APP_DIR} && symfony
 COMPOSER=symfony composer
 CONSOLE=${SYMFONY} console
 export COMPOSE_PROJECT_NAME=cms-page
 export MIGRATIONS_NAMESPACE=MonsieurBiz\\SyliusCmsPagePlugin\\Migrations
-export USER_UID=$(shell id -u)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** I think that this line avoids a warning when the containers are up, so I suggest keeping it‏